app/js/models/userModel.js

```diff
@@ -19,18 +18,19 @@ export default class UserModel extends Model {
    * @extends Model.constructor
    * @param {Object} options
    */
-  constructor(options) {
-    super(options);
-    this.url = options.url;
+  constructor( options ) {
+    super( options );
+    this.url = options.url + '/tokens';
+    this.tenantURL = options.url + '/tenants';
   }
 
   /**
    * Calls setAuthData function.
    * @override Model.parse
    * @param {Object} data
    */
-  parse(data) {
-    this.setAuthData(data);
+  parse( data ) {
```

Comment:
Please use code convention from the project and do not add spaces after and before param.